content/en/docs/3.features/2.deployment-targets.md

```diff
@@ -31,9 +31,9 @@ export default {
 We are also exposing `process.target` for module authors to add logic depending on the user target.
 ::
 
-## Server Hosting
+## Server Hosting (Using Node.js as a webserver backend)
 
-Server Hosting is hosting that requires a server and is intended for SSR applications or applications that are using [serverMiddleware](/docs/configuration-glossary/configuration-servermiddleware). Server-side rendering otherwise known as SSR means that your page is rendered on the server when it is requested by the user. When the user opens your page in a browser the browser sends a request to the server requesting that page. The page is rendered on the server and sent back to the browser with all its content.
+Server Hosting is hosting that requires a Node.js webserver and is intended for SSR applications or applications that are using [serverMiddleware](/docs/configuration-glossary/configuration-servermiddleware). Server-side rendering otherwise known as SSR means that your page is rendered through a Node.js webserver environment when it is requested by the user. When the user opens your page in a browser the browser sends a request to the Node.js webserver requesting that page. The page is rendered on the server and sent back to the browser with all its content.
```

Comment:
```suggestion
Server hosting means running Nuxt on a Node.js server. When the user opens your page, their browser will request that page from the server. Nuxt will handle the request, render the page and send back the resulting page with all its content.

You might need server hosting if you want to render HTML on each request rather than in advance at generate-time, or if you need [serverMiddleware](/docs/configuration-glossary/configuration-servermiddleware). 

::alert{type="info"}
You can still run Nuxt with server hosting with `ssr: false` but Nuxt will not fully render the HTML for each page - leaving that task to the browser. You might choose this option if you need serverMiddleware but do not want fully server-side rendered HTML.
::
```